RecoTracker/LSTCore/src/alpaka/LSTESData.dev.cc

```diff
@@ -17,9 +17,6 @@ namespace {
     std::string path_str;
     if (path_lst_base != nullptr) {
```

Comment:
Maybe we can already deprecate `LST_BASE` since we're probably not planning to run as an external package anymore.

Reply:
`LST_BASE` is now removed.